cmd/helm/install.go

```diff
@@ -181,10 +183,19 @@ func runInstall(args []string, client *action.Install, valueOpts *values.Options
 		return nil, err
 	}
 
-	// Check chart dependencies to make sure all are present in /charts
-	chartRequested, err := loader.Load(cp)
-	if err != nil {
-		return nil, err
+	// Check chart dependencies to make sure all are present in the InMemoryCache or in /charts
+	chartRequested := &chart.Chart{}
```

Comment:
I'm not very experienced in Go, so maybe this does not matter, but it sounds more efficient, instead of initializing to a value that will never be used, tojust declaring the variable like so: `var chartRequested chart.Chart`